kafka-connect/scripts/kafka_connect_run.sh

```diff
@@ -70,6 +70,14 @@ EOF
   # ... but enable equivalent GC logging to stdout
   export KAFKA_GC_LOG_OPTS="-verbose:gc -XX:+PrintGCDetails -XX:+PrintGCDateStamps -XX:+PrintGCTimeStamps"
 
+  if [ -z "$KAFKA_LOG4J_OPTS" ]; then
```

Comment:
Do we want to follow the naming of variables which for Kafka Connect start with `KAFKA_CONNECT_`? I.e. use variable `KAFKA_CONNECT_LOG4J_OPTS to inject the value from outside and only internally map its value to `KAFKA_LOG4J_OPTS`? (I assume this is the name which Kafka Connect has hardcoded inside)

Reply:
Yes I agree with @scholzj

Reply:
Just thinking more, the only part we want to be set by user is the `connect.root.logger` env var right ? Because we don't want having the user changing the properties file as well or do we ?

Reply:
@ppatierno Well, by injecting your own file you would be able to configure not only the log level but for example completely change which log messages you are interested in. So it might be useful. On the other hand I'm not sure this can be done in a "user friendly" way - it basically just leaves some space for manual customization.

Reply:
I don't know if we should provide to the user such level of personalization. Maybe just the log levels are enough. Our objective is running Kafka on OpenShift in a "managed" way so giving too much of flexibility in some case could not be useful, maybe logging is one of these cases. But it's just my opinion of course :-)

Reply:
Actually, thinking about it again - everyone who wants to change the logging completely can just inject his own `$KAFKA_HOME/config/connect-log4j.properties` file through a volume mount. So overridding just the log level through the variable should be sufficient.